src/libstd/sys/vxworks/fd.rs

```diff
@@ -58,19 +58,13 @@ impl FileDesc {
     }
 
     pub fn read_at(&self, buf: &mut [u8], offset: u64) -> io::Result<usize> {
```

Comment:
This function is expected to be thread-safe, this implementation of it is not. To implement this function you will need locking somewhere.